---
layout: page
title: Biography
---
	<section>
		<h2>Junyoung O. Park <a itemprop="sameAs" content="https://orcid.org/0000-0001-9869-8993" href="https://orcid.org/0000-0001-9869-8993" target="orcid.widget" rel="me noopener noreferrer" style="vertical-align:top;"><img src="https://orcid.org/sites/default/files/images/orcid_16x16.png" style="width:1em" alt="ORCID iD icon"></a></h2>
		
		<p>
		<hr />
		<h3>Education</h3>
		<a href="https://www.princeton.edu">Princeton University</a>, Princeton, NJ – <i>2010 - 2016</i>
		<li>Doctor of Philosophy in <i>Chemical and Biological Engineering</i> – <i>2016</i></li>
		<li>Master of Arts in <i>Chemical and Biological Engineering</i> – <i>2012</i></li>
		<br>
		<a href="https://www.ucsd.edu">University of California, San Diego (UCSD)</a>, La Jolla, CA – <i>2007 - 2010</i>
		<li>Bachelor of Science in <i>Bioengineering: Biotechnology</i></li>
		<li>Bachelor of Arts in <i>Mathematics</i></li>
		</p>
		
		<p>
		<h3>Employment</h3>
		<a href="http://www.ucla.edu/">University of California, Los Angeles (UCLA)</a>, Los Angeles, CA – <i>2018 - present</i><br>
		<li>Assistant Professor, Department of Chemical and Biomolecular Engineering</li>
		<li>Co-Director, Metabolomics Center</li>
		<li>Affiliate Faculty, Department of Bioengineering</li>
		<li>Associated Faculty, Institute for Quantitative and Computational Biosciences (QCBio)</li>
		<li>Member, NSF BioPACIFIC MIP</li>
		<li>Member, Molecular Biology Institute (MBI)</li>
		<li>Member, California NanoSystems Institute (CNSI)</li>
		<li>Member, Jonsson Comprehensive Cancer Center (JCCC)</li>
		<li>Member, Goodman-Luskin Microbiome Center (GLMC)</li>
		<br>
		<a href="http://web.mit.edu">Massachusetts Institute of Technology</a>, Cambridge, MA – <i>2016 - 2018</i><br>
		<li>Postdoc, Department of Chemical Engineering</li>
		</p>
		
		<hr />
		<h3>Training</h3>
		Bioinformatics and Metabolic Engineering Laboratory, MIT – <i>2016 - 2018</i><br />
		Adviser: Prof. Gregory Stephanopoulos
		<ul>
		<li>Developed a <sup>13</sup>C-<sup>18</sup>O-based metabolic flux and free energy analysis framework for various cell types including bacteria and cancer cells.</li>
		<li>Applied the metabolic flux and free energy analysis to maximize conversion of CO<sub>2</sub> into oil using <i>Moorella thermoacetica</i> and <i>Yarrowia lipolytica</i>.</li>
		<li>Characterized renal cell carcinoma metabolism under combinatorial therapeutics.</li>
		</ul>
		Lewis-Sigler Institute for Integrative Genomics, Princeton – <i>2010 - 2016</i><br />
		Adviser: Prof. Joshua D. Rabinowitz
		<ul>
		<li>Developed a flux-based method and software to measure Gibbs free energy of metabolic reactions.</li>
		<li>Developed an integrative framework combining metabolite concentrations, fluxes, and thermodynamics for more precise and comprehensive characterization of metabolism.</li>
		<li>Applied the integrated approach to several biological systems including <i>E. coli</i>, yeast, <i>Clostridium</i> species, hepatocytes, renal epithelial cells, and <i>in vivo</i> mouse models to elucidate fundamental metabolic regulation, to achieve effective biofuel production, and to better understand diabetes and cancer.</li>
		<li>Contributed to a successful Department of Energy grant proposal.</li>
		</ul>
		Bio-inspired Materials and Stem Cell Engineering Laboratory, Bioengineering, UCSD – <i>2009 - 2010</i><br />
		Adviser: Prof. Shyni Varghese
		<ul>
		<li>Designed and developed a small diameter hydrogel-based vascular graft that is highly biocompatible, reduces significantly the risk of thrombosis, and increases the patency once placed within patients.</li>
		<li>Conducted various mechanical testing and reached the final design of poly(ethylene glycol) diacrylate (PEGDA)-clay nanocomposite hydrogel, which showed robust mechanical properties that can be controlled by varying clay concentrations.</li>
		</ul>
		Molecular Systems and Gene Regulation Group, Nanoengineering, UCSD – <i>2008 - 2010</i><br />
		Adviser: Prof. Gaurav Arya
		<ul>
		<li>Built a computer model to simulate random walks of transcription factors in the presence of various conformations of compact chromatin fibers using C++.</li>
		<li>Performed kinetic Monte Carlo simulation, which revealed different modes of diffusion that transcription factors undergo: 3D-diffusion, when they are far away from DNA/chromatin fiber and 1D-diffusion along the DNA, otherwise.</li>
		</ul>
		Systems Biology Research Group, Bioengineering, UCSD – <i>2008 - 2009</i><br />
		Adviser: Prof. Bernhard Ø. Palsson
		<ul>
		<li>Developed <a href="http://bigg1.ucsd.edu/">the Biochemical, Genetic and Genomic (BiGG) knowledgebase</a>, which provides the first collection of curated, high quality metabolic reconstructions.</li>
		<li>Integrated metabolic flux constraints as well as objective functions required for mathematical optimization to various organisms’ genome-scale metabolic reconstruction models.</li>
		<li>Rendered respective organisms’ metabolic networks in Scalable Vector Graphics (SVG) that can be interactively displayed in web browsers using Perl and SQL.</li>
		</ul>
		
		<hr />
		<h3>Teaching</h3>
		Department of Chemical and Biomolecular Engineering, UCLA – <i>2018 - present</i>
		<li>Systems Biology: Intracellular Network Identification and Analysis (CBE C146/C246)</li>
		<li>Molecular Biotechnology for Engineers (CBE/BE CM145/CM245)</li>
		<li>Molecular Biotechnology: From Gene to Product (CBE 104D)</li>
		<li>Chemical Engineering Laboratory (CBE 104B)</li>
		<li>Thermodynamics (CBE 200 & CBE 102A)</li>
		<br>
		Department of Chemical Engineering, MIT – <i>2017</i>
		<li>Downstream Processing</li>
		<br>
		Department of Chemical and Biological Engineering, Princeton University – <i>2012 - 2015</i>
		<li>Separations in Chemical Engineering and Biotechnology</li>
		<li>Chemical Engineering Laboratory</li>
		<li>Ethics and Technology: Engineering in the Real World</li>
		<br>
		Seoul Language Institute, South Korea – <i>2010</i>
		<li>Chemistry, English, and Mathematics Instructor</li>
		<br>
		Department of Physics, UCSD – <i>2008</i>
		<li>Reader</li>
		
		<hr />
		<p>
		<h3>Integration of Research and Education</h3>
		<ul class="ul1">
		<li>Mentor, Summer Undergraduate Research Program (SURP), Samueli Research Scholars (SRS) Program, Amazon Summer Undergraduate Research Experience (SURE), and Integrated and Interdisciplinary Undergraduate Research Program (i<sup>2</sup>URP) – <i>2020 - present</i></li>
		<li>Faculty advisor, AIChE UCLA chapter – <i>2020 - present</i></li>
		<li>Mentor, Integrated and Interdisciplinary Undergraduate Research Program (i<sup>2</sup>URP) – <i>2020 - 2022</i></li>
		<li>Mentor, Bruins-In-Genomics (B.I.G.) Summer Undergraduate Research Program (NSF and NIH) – <i>2020 - 2021</i></li>
		<li>Mentor, NSF-LSAMP (NSF-CAMP) – <i>2020 - 2021</i></li>
		<li>CityLab at UCLA – <i>2019</i></li>
		<li>Engineering FT<sup>2</sup> (Faculty Teaching Faculty about Teaching) – <i>2018</i></li>
		<li>National Research Mentoring Network (NRMN) Entering Mentoring Training – <i>2018</i></li>
		<li>Mentor, NetPals STEM Program – <i>2016 - 2017</i></li>
		</ul>
		</p>
		
		<hr />
		<p>
		<h3>Honors</h3>
		NIH Maximizing Investigators' Research Award (MIRA) (R35) – <i>2021</i><br>
		Hellman Fellowship – <i>2020</i><br>
		UCLA Faculty Career Development Award – <i>2020</i><br>
		Professor of the Year (AIChE at UCLA) – <i>2019</i><br>
		Schowalter Travel Fund – <i>2012 - 2015</i><br>
		Hughes Scholars Program at UCSD – <i>2009 - 2010</i><br>
		Warren College Honors Student – <i>2008 - 2010</i><br>
		Provost Honors – <i>2007 - 2010</i><br>
		Otto Schaler Scholarship – <i>2009</i><br>
		Calit2 Summer Undergraduate Research Scholarship – <i>2008 - 2009</i><br>
		</p>
		
		<hr />
		<p>
		<h3>Presentations</h3>
		<ul class="ul1">
		<li>Institute for Quantitative and Computational Biosciences, UCLA. <i>6/11/2025</i></li>
		<li>Gene and Linda Voiland School of Chemical Engineering and Bioengineering, Washington State University. <i>4/7/2025</i></li>
		<li>International SynBYSS Conference. <i>Honolulu, HI, 12/9/2024</i></li>
		<li>Korea-U.S. Emerging Tech Conference. <i>Los Angeles, CA, 11/22/2024</i></li>
		<li>2024 AIChE Annual Meeting. <i>San Diego, CA, 10/29/2024</i></li>
		<li>System Network Inflammation Control Research Center, Chungnam National University College of Medicine. <i>10/8/2024</i></li>
		<li>Korea-U.S. Synthetic Biology Conference. <i>Daejeon, South Korea, 10/7/2024</i></li>
		<li>2024 Research Updates in Lymphoma, Cedars-Sinai Medical Center. <i>9/28/2024</i></li>
		<li>Department of Biological Chemistry, UCI School of Medicine. <i>4/10/2024</i></li>
		<li>Chungnam National University College of Medicine. <i>3/27/2024</i></li>
		<li>BioNano Research Institute, Gachon University. <i>3/25/2024</i></li>
		<li>Department of Chemical and Biological Engineering, Seoul National University. <i>3/22/2024</i></li>
		<li>Department of Chemical Engineering, IIT Delhi. <i>3/1/2024</i></li>
		<li>Mork Family Department of Chemical Engineering and Materials Science, University of Southern California. <i>2/20/2024</i></li>
		<li>2023 AIChE Annual Meeting. <i>Orlando, FL, 11/8/2023</i></li>
		<li>Metabolomics, Metabolic Engineering, and Carbon Upcycling. <i>U.S.- Korea Synthetic Biology Conference, Lawrence Berkeley National Laboratory, Emeryville, CA, 11/3/2023</i></li>
		<li>Metabolomics-Guided Discovery of Metabolic Design Principles and Engineering Strategies. <i>MB 610 Advanced Topics of Bioethics, Bioinformatics, and Molecular Biology, San Diego State University, 10/25/23</i></li>
		<li>Metabolomics-Guided Discovery of Metabolic Design Principles and Engineering Strategies. <i>Metabolic Engineering 15 (ME15), Singapore, 6/11/2023</i></li>
		<li>Uncovering Metabolic Design Principles and Engineering Strategies via Metabolomics. <i>Keck School of Medicine of USC, 4/12/2023</i></li>
		<li>From Basic Design Principles to Metabolic Engineering: Rewiring Central Carbon Metabolism for CO<sub>2</sub> Conversion. <i>Synthetic Biology Young Speaker Series (SynBYSS), 3/30/2023</i></li>
		<li>From Basic Design Principles to Metabolic Engineering: Rewiring Central Carbon Metabolism for CO<sub>2</sub> Conversion. <i>Stevens Institute of Technology, 9/30/2022</i></li>
		<li>Elucidating Glycolytic Flux Control using Metabolomics and Isotope Tracers. <i>UCLA-Caltech Medical Scientist Training Program (MSTP), 2/8/2021</i></li>
		<li>Integrating kinetics, thermodynamics, and metabolomics. <i>Korea Institute for Advanced Study (KIAS) in Seoul, South Korea, 4/30/2019</i></li>
		<li>Integration of metabolomics and fluxomics via thermodynamics. <i>VIB Metabolomics Data Interpretation in Leuven, Belgium, 3/11/2019</i></li>
		<li>Integrative-omics in biotechnology. <i>ISPE Biopharmaceutical Manufacturing Conference in Huntington Beach, 12/11/2018</i></li>
		<li>Glycolysis balances enzyme efficiency, energy efficiency, and metabolic adaptivity. <i>UCLA Metabolism Interest Group, 12/7/2018</i></li>
		<li>Systems-level engineering of metabolic networks. <i>UCLA QCBio Retreat at Calamigos Ranch, Malibu, 9/25/2018</i></li>
		<li>Integration of metabolomics and fluxomics via thermodynamics. <i>Pfizer, Cambridge, 2018</i></li>
		<li>Systems-level engineering of metabolic networks. <i>UCLA, 2018</i></li>
		<li>Efficient energy utilization in carbon-fixing <i>Moorella thermoacetica</i>. <i>AIChE Annual Meeting in Minneapolis, 2017</i></li>
		<li>Park, J. O., Wei, M. H., Tanner, L. B., Amador-Noguez, D., Li, S. H., Rabinowitz, J. D.  Glycolysis balances enzyme efficiency and metabolic adaptivity. <i>Department of Energy Genomic Sciences Program Annual PI Meeting ,2017</i></li>
		<li>Dash, S., Khodayari, A., Zhou, J., Holwerda, E. K., Olson, D. G., Park, J. O., Emerson, D., Woolston, B., Chen, P., Liao, J. C., Stephanopoulos, G., Lynd, L. R., Maranas, C. D. Ensemble cell-wide kinetic modeling of anaerobic organisms to support fuels and chemicals production. <i>Department of Energy Genomic Sciences Program Annual PI Meeting 2017</i></li>
		<li>Towards integrative metabolomics in biotechnology and medicine. <i>Y-IBS symposium at Harvard, 2016</i></li>
		<li>Park, J. O., Amador-Noguez, D., Wei, M. H., Rabinowitz, J. D.  Limited forward thermodynamic driving force in glycolysis of slow-growing bacteria. <i>Department of Energy Genomic Science Contractor-Grantee Meeting XIV 2016</i></li>
		<li>Metabolite concentrations and ΔG that favor efficient enzyme usage are a conserved feature of cellular metabolism. <i>MIT, 2015</i></li>
		<li>Park, J. O., Rubin, S. A., Xu, Y. F., Amador-Noguez, D., Fan, J., Rabinowitz, J. D.  Conservation of absolute cellular metabolite concentrations. <i>Department of Energy Genomic Science Contractors-Grantees Meeting XIII, 2015</i></li>
		<li>Park, J. O., Rubin, S. A., Xu, Y. F., Amador-Noguez, D., Fan, J., Rabinowitz, J. D.  Comprehensive quantitation of metabolite concentrations, gross reaction fluxes, and reaction free energies across cells from four organisms. <i>Metabolism, Diet and Disease, 2014</i></li>
		<li>Amador-Noguez, D., Park, J. O., Zhang, Z., Rubin, S. A., Xu, W., Rabinowitz, J. D.  A fully reversible, highly energy efficient glycolysis with unique cofactor utilization in C. cellulolyticum. <i>Department of Energy Genomic Science Contractor-Grantee Meeting XII, 2014</i></li>
		<li>Xu, Y. F., Doucette, C. D., Park, J. O., Roquet, N., Amador-Noguez, D., Reaves, M. L., Feng, X. J., Rabitz, H., Schwab, D. J., Wingreen, N. S., Rabinowitz, J. D.  Integration of carbon, nitrogen, and oxygen metabolism in Escherichia coli. <i>Department of Energy Genomic Science Awardee Meeting X, 2012</i></li>
		<li>Park, J. O., Xu, Y. F., Rabinowitz, J. D.  Integration of carbon, nitrogen, and oxygen metabolism in Escherichia coli. <i>International Conference and Exhibition on Metabolomics & Systems Biology, 2012</i></li>
		<li>Park, J. O., Arya G.  Facilitated Diffusion of Transcription Factors in Search of DNA Target Sites. <i>Calit2 Summer Scholars Poster Session at UCSD, 2009</i></li>
		<li>Park, J. O., Schellenberger J., Conrad, T. M., Palsson, B. Ø.  BiGG: Biochemical, Genetic and Genomic Database. <i>Calit2 Summer Scholars Poster Session at UCSD, 2008</i></li>
		<li>Metabolic Engineering Guest Lecture. <i>CBE 447, 04/2015</i></li>
		<li>Princeton Chemical Biology Meeting. <i>04/2015, 06/2013</i></li>
		<li>Princeton E-ffiliates Partnership Retreat. <i>02/2015</i></li>
		<li>Princeton Prokaryotes Meeting. <i>03/2016, 11/2015, 10/2014, 05/2014, 10/2013, 11/2012</i></li>
		<li>Princeton BioEngineering Colloquium. <i>03/2014</i></li>
		<li>Graduate Student Symposium. <i>10/2013, 10/2012</i></li>
		</ul>
		</p>

	</section>
	
<!--		<p>
		<h3>Skills</h3>
		<b>Excellent interpersonal, project management and leadership skills</b><br>
		<b>Analytical chemistry</b>: Mass spectrometry, High-performance liquid chromatography, LC/MS, GC/MS, Nuclear magnetic resonance spectroscopy<br>
		<b>Computer</b>: C/C++, Perl, Python, PHP, MATLAB, Mathematica, R, Fortran, Java, JavaScript, SQL, HTML, CSS, Systems Biology Markup Language (SBML), LaTeX, SVG, PARI/GP, VMD; Apache HTTP Server, MySQL, SVN, Git, OpenMP, MPI; MS Excel, PowerPoint, Word, Adobe Illustrator, LabVIEW; Linux, Windows, Mac<br>
		<b>Languages</b>: English, Korean (fluent); Spanish, Japanese, Latin (basic); Chinese (basic reading)<br>
		<b>Musical instruments</b>: Piano, Guitar, Trombone<br>
		<b>Sports</b>: Tennis, Golf, Swimming, Soccer, Softball, Skiing, Sailing, Rowing<br>
		</p>
		<hr />
		<p>
		<h3>Honors</h3>
		USC Provost’s PhD Fellowship (declined) – <i>2010</i><br>
		Hughes Scholars Program at UCSD – <i>2009 - 2010</i><br>
		Warren College Honors Student – <i>2008 - 2010</i><br>
		Provost Honors – <i>2007 - 2010</i><br>
		Otto Schaler Scholarship – <i>2009</i><br>
		Calit2 Summer Undergraduate Research Scholarship – <i>2008 - 2009</i><br>
		</p>
		<hr />
		<p>
		<h3>Leadership & Professional Experience</h3>
		Cell Chemical Biology reviewer – <i>2017 - present</i><br>
		Nature Communications reviewer – <i>2016 - present</i><br>
		Metabolic Engineering reviewer – <i>2015 - present</i><br>
		Physical Biology reviewer – <i>2015 - present</i><br>
		Journal of Microbiology and Biotechnology reviewer – <i>2015 - present</i><br>
		PLOS ONE reviewer – <i>2015 - present</i><br>
		MIT Club Golf – <i>2016 - present</i><br>
		Captain of Chemical Engineering Intramural Tennis Team – <i>2016</i><br>
		Chemical and Biological Engineering Softball Team – <i>2011 - 2014</i><br>
		Princeton Club Tennis – <i>2011 - 2014</i><br>
		Princeton University Wind Ensemble – <i>2011</i><br>
		Elected Member of GSC, the Graduate Student Committee – <i>2010 - 2011</i><br>
		Bioengineering Senior Design Project – <i>2009 - 2010</i><br>
		TESC, the Triton Engineering Student Council – <i>2007 - 2008</i><br>
		</p>
		<hr />
		<p>
		<a href="publications"><h3>Publications</h3></a>
		</p>
		<hr />
	
 	<section>
		<h4>Image</h4>
		<h5>Fit</h5>
		<div class="box alt">
			<div class="row uniform 50%">
				<div class="12u"><span class="image fit"><img src="images/banner.jpg" alt="" /></span></div>
				<div class="4u"><span class="image fit"><img src="images/pic01.jpg" alt="" /></span></div>
				<div class="4u"><span class="image fit"><img src="images/pic02.jpg" alt="" /></span></div>
				<div class="4u"><span class="image fit"><img src="images/pic03.jpg" alt="" /></span></div>
				<div class="4u"><span class="image fit"><img src="images/pic03.jpg" alt="" /></span></div>
				<div class="4u"><span class="image fit"><img src="images/pic02.jpg" alt="" /></span></div>
				<div class="4u"><span class="image fit"><img src="images/pic01.jpg" alt="" /></span></div>
				<div class="4u"><span class="image fit"><img src="images/pic02.jpg" alt="" /></span></div>
				<div class="4u"><span class="image fit"><img src="images/pic01.jpg" alt="" /></span></div>
				<div class="4u"><span class="image fit"><img src="images/pic03.jpg" alt="" /></span></div>
			</div>
		</div>
		<h5>Left &amp; Right</h5>
		<p><span class="image left"><img src="images/pic04.jpg" alt="" /></span>Morbi mattis mi consectetur tortor elementum, varius pellentesque velit convallis. Aenean tincidunt lectus auctor mauris maximus, ac scelerisque ipsum tempor. Duis vulputate ex et ex tincidunt, quis lacinia velit aliquet. Duis non efficitur nisi, id malesuada justo. Maecenas sagittis felis ac sagittis semper. Curabitur purus leo, tempus sed finibus eget, fringilla quis risus. Maecenas et lorem quis sem varius sagittis et a est. Maecenas iaculis iaculis sem. Donec vel dolor at arcu tincidunt bibendum. Interdum et malesuada fames ac ante ipsum primis in faucibus. Fusce ut aliquet justo. Donec id neque ipsum. Integer eget ultricies odio. Nam vel ex a orci fringilla tincidunt. Aliquam eleifend ligula non velit accumsan cursus. Etiam ut gravida sapien. Morbi mattis mi consectetur tortor elementum, varius pellentesque velit convallis. Aenean tincidunt lectus auctor mauris maximus, ac scelerisque ipsum tempor. Duis vulputate ex et ex tincidunt, quis lacinia velit aliquet. Duis non efficitur nisi, id malesuada justo. Maecenas sagittis felis ac sagittis semper. Curabitur purus leo, tempus sed finibus eget, fringilla quis risus. Maecenas et lorem quis sem varius sagittis et a est. Maecenas iaculis iaculis sem. Donec vel dolor at arcu tincidunt bibendum. Interdum et malesuada fames ac ante ipsum primis in faucibus. Fusce ut aliquet justo. Donec id neque ipsum. Integer eget ultricies odio. Nam vel ex a orci fringilla tincidunt. Aliquam eleifend ligula non velit accumsan cursus. Etiam ut gravida sapien.</p>
		<p><span class="image right"><img src="images/pic05.jpg" alt="" /></span>Vestibulum ultrices risus velit, sit amet blandit massa auctor sit amet. Sed eu lectus sem. Phasellus in odio at ipsum porttitor mollis id vel diam. Praesent sit amet posuere risus, eu faucibus lectus. Vivamus ex ligula, tempus pulvinar ipsum in, auctor porta quam. Proin nec dui cursus, posuere dui eget interdum. Fusce lectus magna, sagittis at facilisis vitae, pellentesque at etiam. Quisque posuere leo quis sem commodo, vel scelerisque nisi scelerisque. Suspendisse id quam vel tortor tincidunt suscipit. Nullam auctor orci eu dolor consectetur, interdum ullamcorper ante tincidunt. Mauris felis nec felis elementum varius. Nam sapien ante, varius in pulvinar vitae, rhoncus id massa. Donec varius ex in mauris ornare, eget euismod urna egestas. Etiam lacinia tempor ipsum, sodales porttitor justo. Aliquam dolor quam, semper in tortor eu, volutpat efficitur quam. Fusce nec fermentum nisl. Aenean erat diam, tempus aliquet erat. Etiam iaculis nulla ipsum, et pharetra libero rhoncus ut. Phasellus rutrum cursus velit, eget condimentum nunc blandit vel. In at pulvinar lectus. Morbi diam ante, vulputate et imperdiet eget, fermentum non dolor. Ut eleifend sagittis tincidunt. Sed viverra commodo mi, ac rhoncus justo. Duis neque ligula, elementum ut enim vel, posuere finibus justo. Vivamus facilisis maximus nibh quis pulvinar. Quisque hendrerit in ipsum id tellus facilisis fermentum. Proin mauris dui.</p>
	</section>
 -->