---
layout: page
title: Integrative Metabolomics
categories: hidden
---
	<h3>Integration of metabolomics and fluxomics via nonequilibrium thermodynamics</h3>
	<p>Metabolism is the process of converting nutrients into usable energy (e.g., ATP) and biomass building blocks (e.g., amino acids, nucleotides, and lipids). The biochemistry of metabolic reactions and the structure of metabolic networks bear impressive resemblance across widely divergent organisms. Like all chemical networks, metabolism must obey the second law of thermodynamics: each pathway step must generate entropy and cost free energy (&Delta;G). As available free energy is limited, a fundamental challenge is partitioning the requisite free energy loss across pathway steps.</p>
	<p>For each reaction, &Delta;G is log-proportional both to a concentration ratio (reaction quotient-to-equilibrium constant) and to a flux ratio (backward-to-forward flux). With <sup>13</sup>C isotope labeling, absolute metabolite concentrations and fluxes can be measured in various cell types and organisms including <i>E. coli</i>, yeast, and mammalian cells. Then, the concentration and flux ratios can be integrated via &Delta;G using the non-equilibrium thermodynamic principle. The integrative analysis yields internally consistent and comprehensive sets of metabolite concentrations and &Delta;G in each organism. In glycolysis, free energy is partitioned so as to mitigate unproductive backwards fluxes associated with &Delta;G near zero. Across metabolism, absolute metabolite concentrations and &Delta;G are substantially conserved such that most substrate (but not inhibitor) concentrations exceed the associated enzyme binding site affinity. The observed conservation of metabolite concentrations may reflect an evolutionary drive to simultaneously satisfy thermodynamic constraints and efficiently utilize enzyme active sites.</p>
	<p><a href="research#integrative_metabolomics">Back to </a><a href="research">Research</a></p>
	<hr />
	<p>
	<h3>References</h3>
	<ul class="ul1">
	<li>Park, J. O., Rubin, S. A., Xu, Y. F., Amador-Noguez, D., Fan, J., Shlomi, T., Rabinowitz, J. D.  <a href="https://www.nature.com/nchembio/journal/v12/n7/abs/nchembio.2077.html" onclick="trackOutboundLink('https://www.nature.com/nchembio/journal/v12/n7/abs/nchembio.2077.html'); return false;">Metabolite concentrations, fluxes and free energies imply efficient enzyme usage.</a> <i>Nature Chemical Biology</i> 12:482-489 (2016) doi: 10.1038/nchembio.2077</li>
	<li>Schellenberger, J., Park, J. O., Conrad, T. M., Palsson, B. Ã˜.  <a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-11-213" onclick="trackOutboundLink('https://bmcbioinformatics.biomedcentral.com/articles/10.1186/1471-2105-11-213'); return false;">BiGG: A Biochemical Genetic and Genomic knowledgebase of large scale metabolic reconstructions.</a> <i>BMC Bioinformatics</i> 11:213 (2010) doi: 10.1186/1471-2105-11-213</li>
	</ul>
	</p>