---
layout: page
title: Research
---
	<h3>From integrative metabolomics to understanding human diseases and enhancing CO<sub>2</sub> fixation </h3>
	<p>A growing concern for 21st-century humanity is the irrevocable rise of carbon dioxide (CO<sub>2</sub>). On a more personal level, many individuals today suffer from distressing diseases like cancer. Potential solutions to these disparate problems lie in metabolism. With these applications in mind, my research goal is to advance biotechnology and medicine by integrative metabolomics, which incorporates kinetic and thermodynamic information into characterization of metabolic processes.</p>
	<p>My postdoctoral research with Prof. Gregory Stephanopoulos has focused on enhancing CO<sub>2</sub> fixation by employing thermodynamic and systems-level efficiency of non-photosynthetic carbon-fixing metabolism. I developed a rapid CO<sub>2</sub> fixation scheme that enables precise control over the rate of CO<sub>2</sub> fixation (i.e., productivity) and the proportion of CO<sub>2</sub> incorporated into the product (i.e., yield). Using this technology, fermentation of <i>Moorella thermoacetica</i> produced acetate as a product of CO<sub>2</sub> fixation at high specific productivities (0.54 g per g cell dry weight (gCDW) per hour) with nearly all acetate carbons (>98%) derived from CO<sub>2</sub>.</p>
	<p>In establishing this technology, my graduate and undergraduate training with Profs. Joshua Rabinowitz and Bernhard Palsson helped with quantitative characterization of metabolism. Using non-equilibrium thermodynamic principles and stoichiometric metabolic models, I was able to integrate forward-to-backward reaction flux ratios with reaction quotient-to-equilibrium constant ratios via Gibbs free energy. The upshot of the integrative analysis was internally consistent and comprehensive sets of metabolite concentrations and Gibbs energies in <i>E. coli</i>, yeast, and mammalian cells. Based on these results, I found efficient enzyme usage to be a design principle across kingdoms of life.</p>
	<p>Collaborating with other researchers, I recognized that there is ample room for new discoveries by applying the integrative metabolomic approaches in other fields. Working with Profs. Mark Brynildsen and Stanislav Shvartsman, I contributed to understanding the underlying metabolic processes in bacterial persistence and in healthy embryonic development. The resulting knowledge pointed to potential drug targets in bacteria with high antibiotic tolerance and to an essential metabolic regulation that ensures normal development.</p>
	<p>Going forward, I would like to incorporate comprehensive enzyme cost and efficiency information into the metabolic flux and pathway analysis paradigm. Lack of efficiency in certain cells (e.g., cancer cells) could suggest potential therapeutic targets. In metabolic engineering applications, the economics of cellular proteome partitioning must be compatible with our economic goals. Optimal and feasible solutions, then, would consist of metabolic modules that simultaneously permit maximum enzyme efficiency and maximum net value created. Such integrative approaches will help disentangle complex metabolic diseases and engineering challenges, thereby facilitating better therapeutic and engineering solutions.</p>
	<hr />
	<h3>Research Experience</h3>
	Bioinformatics and Metabolic Engineering Laboratory, MIT – <i>2016 - present</i><br />
	Adviser: Prof. Gregory Stephanopoulos
	<ul>
	<li>Developed a <sup>13</sup>C-<sup>18</sup>O-based metabolic flux and free energy analysis framework for various cell types including bacteria and cancer cells.</li>
	<li>Applied the metabolic flux and free energy analysis on carbon-fixing <i>Moorella thermoacetica</i>.</li>
	</ul>
	Lewis-Sigler Institute for Integrative Genomics, Princeton – <i>2010 - 2016</i><br />
	Adviser: Prof. Joshua D. Rabinowitz
	<ul>
	<li>Developed a flux-based method and software to measure Gibbs free energy of metabolic reactions.</li>
	<li>Developed an integrative framework combining metabolite concentrations, fluxes, and thermodynamics for more precise and comprehensive characterization of metabolism.</li>
	<li>Applied the integrated approach to several biological systems including <i>E. coli</i>, yeast, <i>Clostridium</i> species, hepatocytes, renal epithelial cells, and <i>in vivo</i> mouse models to elucidate fundamental metabolic regulation, to achieve effective biofuel production, and to better understand diabetes and cancer.</li>
	<li>Contributed to a successful Department of Energy grant proposal.</li>
	</ul>
	Bio-inspired Materials and Stem Cell Engineering Laboratory, Bioengineering, UCSD – <i>2009 - 2010</i><br />
	Adviser: Prof. Shyni Varghese
	<ul>
	<li>Designed and developed a small diameter hydrogel-based vascular graft that is highly biocompatible, reduces significantly the risk of thrombosis, and increases the patency once placed within patients.</li>
	<li>Conducted various mechanical testing and reached the final design of poly(ethylene glycol) diacrylate (PEGDA)-clay nanocomposite hydrogel, which showed robust mechanical properties that can be controlled by varying clay concentrations.</li>
	</ul>
	Molecular Systems and Gene Regulation Group, Nanoengineering, UCSD – <i>2008 - 2010</i><br />
	Adviser: Prof. Gaurav Arya
	<ul>
	<li>Built a computer model to simulate random walks of transcription factors in the presence of various conformations of compact chromatin fibers using C++.</li>
	<li>Performed kinetic Monte Carlo simulation, which revealed different modes of diffusion that transcription factors undergo: 3D-diffusion, when they are far away from DNA/chromatin fiber and 1D-diffusion along the DNA, otherwise.</li>
	</ul>
	Systems Biology Research Group, Bioengineering, UCSD – <i>2008 - 2009</i><br />
	Adviser: Prof. Bernhard Ø. Palsson
	<ul>
	<li>Developed <a href="http://bigg1.ucsd.edu/">the Biochemical, Genetic and Genomic (BiGG) knowledgebase</a>, which provides the first collection of curated, high quality metabolic reconstructions.</li>
	<li>Integrated metabolic flux constraints as well as objective functions required for mathematical optimization to various organisms’ genome-scale metabolic reconstruction models.</li>
	<li>Rendered respective organisms’ metabolic networks in Scalable Vector Graphics (SVG) that can be interactively displayed in web browsers using Perl and SQL.</li>
	</ul>
	</p>